packages/client/src/Tests/mappingHelpers/assetsVisibleToMe.test.js

```diff
@@ -0,0 +1,84 @@
+/* global it expect */
+
+import assetsVisibleToMe from '../../Components/Mapping/helpers/assetsVisibleToMe'
+import { UMPIRE_FORCE } from '../../consts'
+
+/* sample dataset that is used across all tests in this file
+*/
+const allForces = [
```

Comment:
This looks like it's more or less the same as the `allForces` data from the previous test, so this is a good example of how useful making code reusable can be. :)

The differences in code are good as well as it shows that your functions are filtering code as expected.